src/main/java/uk/ac/cam/cl/dtg/segue/api/AuthenticationFacade.java

```diff
@@ -98,6 +99,42 @@ public AuthenticationFacade(final PropertiesLoader properties, final UserAccount
         this.misuseMonitor = misuseMonitor;
     }
 
+    /**
+     * Get authentication provider information for the current user
+     *
+     * @param request
+     *            - the http request containing session information of the user currently logged in
+     * @param userId - the id of the user of interest
+     * @return a user authentication settings object showing information about how a given user can authenticate
+     */
+    @GET
+    @Path("/user_authentication_settings/{user_id}")
```

Comment:
As a top-level endpoint to be used from the My Account page, this isn't really consistent with the other five that get loaded when you visit that page:
`GET /users/current_user`
`GET /users/user_preferences`
`GET /authorisations`
`GET /authorisations/other_users`
`GET /groups/memberships`
`GET /auth/user_authentication_settings/{user_id}`

I think it would be nicer if there was a plain endpoint (`GET /auth/user_authentication_settings`) for just the current user, and leave this one for admin things?